sunpy/net/helio/tests/test_helio.py

```diff
@@ -294,6 +295,26 @@ def test_client_search(client):
     assert len(res) == 10
 
 
+@pytest.mark.remote_data
+def test_client_search_warning(client):
```

Comment:
What I was trying to say is that we don't need the same test twice.

Reply:
Yes, is that not the case with this test?

Reply:
Yes, so it must be removed and just stick with `test_client_search`, right?

Reply:
I would say so, is there a need to have the test repeated?

Reply:
No, I don't think it's possible for `warn_user` not to be raised if we reach `len(table) == max_records`.